migrations/2019-04-18-233116_add_to_action_modifier_enum/down.sql

```diff
@@ -0,0 +1,11 @@
+-- This file should undo anything in `up.sql`
+CREATE TYPE action_modifier_new AS ENUM ('all', 'own');
+
+ALTER TABLE permissions
+  ALTER COLUMN modifier TYPE action_modifier_new[]
+    USING (modifier::text::action_modifier_new[]);
```

Comment:
What happens if you have a column with 'one'?

Reply:
What do you mean by that? This migration shouldn't have to worry about that because `up.sql` doesn't create any new rows right?

Reply:
Imagine we have existing data and we need to roll back for some reason. I'd just remove those fields

Reply:
I'm not sure how to remove the 'one' value from existing rows in the permissions column. Where should I look for information to find out? And/or am I approaching this incorrectly? The only way I can think to do this is

1. Create a new column `modifier_new` with datatype `action_modifier`
2. Copy the data over
    ```sql
    ALTER TABLE permissions
      ALTER COLUMN modifier_new TYPE action_modifier[]
        USING (modifier);
    ```
3. Remove `one` from all instances of the `modifier_new` field
4. Execute
    ```sql
    ALTER TABLE permissions
      ALTER COLUMN modifier_new TYPE action_modifier_new[]
        USING (modifier_new::text::action_modifier_new[]);
    ```
5. Rename stuff and remove original column

Reply:
What you have written out sounds great! It's a little complicated, but migrations are tricky things to write. Let me know if you have any trouble implementing it.